protos/feast/core/FeatureSet.proto

```diff
@@ -16,31 +16,39 @@
 
 syntax = "proto3";
 
-package feast.specs;
+package feast.core;
 
-option java_package = "feast.specs";
+option java_package = "feast.core";
 option java_outer_classname = "FeatureSetProto";
-option go_package = "github.com/gojek/feast/protos/generated/go/feast/specs";
+option go_package = "github.com/gojek/feast/protos/generated/go/feast/core";
 
 import "feast/types/Value.proto";
+import "feast/core/Source.proto";
 
 message FeatureSet {
   // Name of the featureSet. Must be unique.
   string name = 1;
+
   // FeatureSet version.
-  string version = 2;
+  int32 version = 2;
+
   // Features contained within this featureSet.
-  repeated FeatureSpec features = 3;
+  repeated Feature features = 3;
+
+  // Source on which feature rows can be found
+  Source source = 4;
 }
 
-message FeatureSpec {
+message Feature {
```

Comment:
Can this cause ambiguity when we're referring to `Feature`?
Is it referering to `feast.core.Feature` or `feast.types.Feature`?
The later actually refers to a feature with name and value.

Reply:
I agree that it is a bit ambiguous, but I want to get away from using the "spec" suffix. Do you have a suggestion for a better name?

Reply:
I think in this case `spec` is the correct term to use